alvr/launcher/src/actions.rs

```diff
@@ -272,9 +272,11 @@ pub fn data_dir() -> PathBuf {
         PathBuf::from(env::var("HOME").expect("Failed to determine home directory"))
             .join(".local/share/ALVR-Launcher")
     } else {
-        env::current_dir()
+        env::current_exe()
             .expect("Unable to determine executable directory")
-            .join("ALVR-Launcher")
+            .parent()
+            .unwrap()
+            .to_owned()
```

Comment:
Are you sure it's fine to just change this considering updates? Or do you just assume the data directory is fucked anyway if you update the launcher since the user will probably put it into a different path or delete the original launcher directory and replace it?

Reply:
The latter 😅

Reply:
Seems fair enough